server/functions/.prettierrc

```
@@ -0,0 +1,4 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2
```

Include single quotes in prettierc
```suggestion
    "tabWidth": 2,
    "singleQuote": true
```

---

So, to clarify: we're currently using double quotes everywhere. The current `eslint` config is set up to follow `prettier`'s default here. You feel we should switch that around?